src/github/push.ts

```diff
@@ -3,8 +3,10 @@ import { Context } from "probot/lib/context";
 import { getCurrentTime } from "utils/webhook-utils";
 import { hasJiraIssueKey } from "utils/jira-utils";
 import { GitHubPushData } from "../interfaces/github";
+import { WebhookContext } from "../routes/github/webhook/webhook-context";
 
-export const pushWebhookHandler = async (context: Context, jiraClient): Promise<void> => {
+export const pushWebhookHandler = async (probotContext: Context, jiraClient, _util, _githubInstallationId: number, webhookContext?: WebhookContext): Promise<void> => {
+	const context = webhookContext ? webhookContext : probotContext;
```

Comment:
Shouldn't we be removing the dependency on probot for this new handler?

Reply:
Same handler is being used for both old (Probot) and new webhook receiver. When push handler is called with new webhook receiver, `probotContext ` empty whereas `WebhookContext` populated.

Reply:
Sure, but the whole point of the move is to remove probot.  We should try to standardize it to use the same `WebhookContext` for both instead of have the probot context pollute the codebase and making it harder to code because of conditionals.

Reply:
If this is not possible, we should probably just create a new push handler since we have a new entrypoint for webhooks anyways.  Once we move to the new webhook url, we can delete everything probot.

Reply:
@mboudreau, I'm fine with creating a new push handler. We just need to make sure that if we fix any bug in exisiting Probot webhook handler, that needs to be done in new webhook codebase until we move away from Probot.